doc/account.md

```diff
@@ -12,12 +11,16 @@ The `idHash` property of `account` covers how the ids will be hashed when stored
 
 You can find a list of available hashing methods in `lib/ciphers-and-hashes.js`.
 
-Where the account information is stored can be change by updating `accountDir`. The registration files can be updated by changing `registrationDir`. Ideally these would be different as we store temporary registration files during the sign up process in the `registrationDir`, and permanent files for the account in `accountDir` once sign up is complete. These cannot be local directories as they will be passed to [storage] to handle.
-
 The lengths of the IDs we use for account creation can be adjusted using `accountIdLength` and `registrationIdLength`.
 
+There are several ways of managing how login information is created. Using `loginDir` will change where the login file is created. *This must be have a leading and ending slash.* Changing how long the id for login is changed using `loginIdLength`. Lastly how long the login file stays on the server is managed by `loginLifetime`. This should be relatively short as these login files should not stay around too long so we can fight against time attacks.
+
+The `passwordHash` section contains how the  passwords will be hashed when creating a password and logging in. This contains the same information as `idHash` as we use the same technique to hash the passwords just with different properties. The salt will be filled in when sent to the client. **These settings should not be changed after being set up.**
```

Comment:
I've been thinking about this and I want to be able to change the pbkdf2 settings at any time.  This means that the pbkdf2 settings that were used to generate the hashed password (we need to figure out a term for this) should be all encoded into the account record as well.